### PR TITLE
Fix gulp-icons to work with Node 12

### PR DESCRIPTION
**Pull request by web-padawan**
Based on https://github.com/vaadin/vaadin-lumo-styles/pull/88

Also added a demo page to check how the generated icons look like.

### CODE REVIEW

File: demo/icons.html [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+        justify-content: center;
+        width: 2rem;
+        height: 2rem;
+        margin: 0.25rem;
+        color: var(--material-secondary-text-color);
+        background-clip: content-box;
+        position: relative;
+        font-weight: 600;
+        background-color: var(--material-contrast-10pct);
+      }
```
Comment (jouni) [inline demo/icons.html:45]: This is not needed, although it doesn’t really matter either.
```suggestion
```

**Review by jouni — APPROVED** [review]
Tested locally 👍

**Comment by jouni** [comment]
Sorry about taking so long to review.

**Comment by web-padawan** [comment]
Thanks for the review! 🙇

**Pull request merged**